src/GraphQL.Tests/Validation/UniqueDirectivesPerLocationTests.cs

```diff
@@ -67,7 +76,8 @@ public void duplicate_directives_in_one_location()
                     field @directive @directive
                 }
                 ";
-                duplicateDirective(_, "directive", 3, 27, 3, 38);
+                duplicateDirective(_, "directive", 3, 27);
```

Comment:
Note that now there will be as many errors as there are non-repeatable directives.